src/Compilers/Core/Portable/Syntax/SyntaxList.SeparatedWithManyChildren.cs

```diff
@@ -10,24 +12,24 @@ internal class SeparatedWithManyChildren : SyntaxList
         {
             private readonly ArrayElement<SyntaxNode>[] _children;
```

Comment:
💡 Changing this to `ArrayElement<SyntaxNode?>[]` seems to be an improvement for this type